CI/unit_tests/project/test_project_add_experiment.py

```diff
@@ -25,17 +25,63 @@
 -------
 """
 import os
+import pathlib
 
 import MDAnalysis
 import numpy as np
 import pytest
+from ase import units
+from ase.calculators.emt import EMT
+from ase.io import write
+from ase.lattice.cubic import FaceCenteredCubic
+from ase.md.velocitydistribution import MaxwellBoltzmannDistribution
+from ase.md.verlet import VelocityVerlet
 from zinchub import DataHub
 
 import mdsuite as mds
 import mdsuite.file_io.chemfiles_read
 import mdsuite.file_io.lammps_flux_files
 
 
+@pytest.fixture(scope="session")
+def ase_md_extxyz(tmp_path_factory) -> pathlib.Path:
```

Comment:
Need to add the actual test

Reply:
who writes it?

Reply:
I did in [2509418](https://github.com/zincware/MDSuite/pull/501/commits/2509418d471c8659027b5616a36fc29d04897188)